src/main/java/frc/robot/subsystems/hood/HoodConstants.java

```diff
@@ -17,13 +22,18 @@ public class HoodConstants {
     public static final Measure<Mult<Mult<Mass, Distance>, Distance>> MOMENT_OF_INERTIA =
             Units.Kilograms.mult(Units.Meters).mult(Units.Meters).of(0.0003);
```

Comment:
Is this the real value? You can get it from CAD.

Reply:
This is the real value from the CAD

Reply:
Oh, sorry, I expected something larger.